Dockerfile.foreman

```diff
@@ -0,0 +1,18 @@
+FROM python:3.5-slim
+
+RUN groupadd user && useradd --create-home --home-dir /home/user -g user user
+WORKDIR /home/user
+
+ENV CELERY_VERSION 4.0.2
+
+COPY workers/requirements.in .
+
+RUN pip install pip-tools && \
+    pip-compile requirements.in && \
+    pip install -r requirements.txt
```

Comment:
The `requirements.txt` file is actually generated by `pip-compile`. I think I'm doing this wrong here. I should be running `pip-compile` as a pre-deployment step and then be keeping the resulting `requirements.txt` under source control so that as `pip-compile` changes version numbers the changes are tracked. Then in these dockerfiles I should just be installing from the `requirements.txt` without compiling as part of building the container.